### ISSUE TITLE
ambientMaterial reference

### ISSUE THREAD

**Issue by mbmosher** [processing/p5.js#6115]
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [X] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### p5.js version

1.6.0

### Steps:
1. Open reference for ambient material [https://p5js.org/reference/#/p5/ambientMaterial](url)
2. Read first line, text in bold "The ambientMaterial() color is the color the object will reflect under **any** lighting."
3. This is false, ambientMaterial only interacts with ambientLights.  It has no bearing on directional and point lights.

I believe this is on line 724 of the reference doc. [https://github.com/processing/p5.js/blob/v1.6.0/src/webgl/material.js#L720](url)

**Comment by davepagurek** [comment]
Makes sense! Maybe we can replace it with something like "The ambientMaterial() color represents the components of the ambientLight() color that the object reflects"

**Comment by ArshM17** [comment]
Also the following lines should state "If the ambientLight() emits the color white..." instead of just "If the light emits the color white...". May I do the necessary changes and make a PR?
